src/client/containers/Home/Home.styles.css

```diff
@@ -208,6 +208,7 @@
     display: block;
     margin-top: 50%;
     padding-top: 1%;
+    padding-left: 2rem;
```

Comment:
Can you add a bit of padding at the bottom? Something like `padding-bottom: 2em`;